docs/source/format/CanonicalExtensions.rst

```diff
@@ -251,6 +251,25 @@ Variable shape tensor
    Values inside each **data** tensor element are stored in row-major/C-contiguous
    order according to the corresponding **shape**.
 
+.. _json_extension:
+
+JSON
+====
+
+* Extension name: `arrow.json`.
+
+* The storage type of this extension is ``StringArray`` or
+  ``LargeStringArray``or ``StringViewArray``.
+
+* Extension type parameters:
+
+  This type does not have any parameters.
+
+* Description of the serialization:
+
+  Metadata is an empty string.
+  - Example: ``""``
```

Comment:
I don't think the example is useful ;-)

Reply:
It's great. It covers all cases. Removing.